app/tests/integration/utils/filter_expression.js

```diff
@@ -0,0 +1,22 @@
+Components.utils.import("resource://gre/modules/components-utils/FilterExpressions.jsm");
+Components.utils.import("resource://gre/modules/components-utils/ClientEnvironment.jsm");
+Components.utils.import("resource://messaging-system/lib/RemoteSettingsExperimentLoader.jsm");
+
+// Targeting
+
+async function remoteSettings(arguments) {
+
+    const { TargetingContext } = ChromeUtils.import(
+        "resource://messaging-system/targeting/Targeting.jsm"
+    );
+    const ENVIRONMENT = {version: arguments[0], localeLanguageCode: "en-US"};
+    let targeting = new TargetingContext(ENVIRONMENT);
+    const EXPRESSION = `version|versionCompare('${arguments[1]}.!') >= 0 && localeLanguageCode == 'en'`;
+    console.log(EXPRESSION);
```

Comment:
Probably can ✂️ this debuggery.

Reply:
> Lookin' good!
> Are these tests running in CI somewhere yet?

They're running in the integration job here. Take a look at the "checks" section on github.